### PR TITLE
ui testing scheleton

### PR DESCRIPTION
**Pull request by iyatan**

### CODE REVIEW

**Review by preyansh98 — APPROVED** [review]
LGTM! Can we also add package-lock.json to .gitignore?

**Review by nath0000000 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules
+tests_output
+*.log
+
```
Comment (ahmedazz260) [inline .gitignore]: please add  to the list: 
package-lock.json

**Review by ahmedazz260 — DISMISSED** [review]
Good job,

just be sure to add package-lock.json like preyansh mentioned

**Comment by preyansh98** [comment]
@ahmedazz260 you can merge this in as the new changes have been done

**Pull request merged**